charts/controller/README.md

```diff
@@ -57,10 +57,23 @@ Those commands removes all the Kubernetes components associated with the chart a
 
 ## Configuration
 
+### Server Modes
+
+The controller runs with one of 3 modes (controller.serverMode):
+
+* **TLS** - If you have your own certificate that resides also for your teammates browser, you will need to create :
```

Comment:
The reason for TLS here isn't like our DOP.
The cert doesn't need to reside in the users browser, as they use it only for the sdk setup.